mkdocs.yml

```diff
@@ -1,7 +1,7 @@
 site_name: Inteligentne Podlewanie Roślin
```

Comment:
tu korzustajmy z material,jest lepsze

jak chcesz light mode, wrzuc mi na to taska

Reply:
nie działał mi mkdocs i zapomniałem to zmienić